pathfinder/paizo/adventure_path/ruins_of_azlant/players_guide/roapg_abilities.lst

```diff
@@ -0,0 +1,22 @@
+# CVS $Revision: $ $Author: $ -- Mon Sep 15 21:55:54 2014 -- reformated by prettylst.pl v1.51 (build 25129)
+SOURCELONG:Ruins of Azlant Player's Guide	SOURCESHORT:ROAPG	SOURCEWEB:http://paizo.com/products/btpy9uir?Pathfinder-Adventure-Path-Ruins-of-Azlant-Players-Guide	SOURCEDATE:2017-08
+# Original Entry by: Stefan Radermacher
+
+###Block: Campaign Traits
+# Ability Name	Product Identity?	Desc Product Identity	Key	Category of Ability	Type	Multiple Requirements	Description	Stackable?	Multiple?	Choose	Save bonus	Bonus to skill	Class Skill	Bonus Language	Source Page	Bonus												
+Athletic		DESCISPI:YES	KEY:Trait ~ Athletic	CATEGORY:Special Ability	TYPE:Trait.CampaignTrait	PREMULT:1,[PREABILITY:1,CATEGORY=Special Ability,Trait ~ Athletic],[PREVAREQ:BypassTraitRestriction,1],[!PREABILITY:1,CATEGORY=Special Ability,TYPE.CampaignTrait]	DESC:You have always had a knack for physical activity. You either grew up in a rural area and had an active childhood where you were always climbing trees, swimming in lakes and rivers, and clambering up rocky hillsides, or you competed in and excelled at contests or sports while growing up in a larger settlement. Due to this background, little can slow you down or inhibit your movements. When the Bountiful Venture Company interviewed you, they noticed your stature and physicality, and they chose you for the colonial expedition knowing that a strong back is always helpful in a growing colony. You reduce your armor check penalty by a number equal to one-third your character level (minimum 1) for purposes of Acrobatics, Climb, and Swim checks, and you gain a +1 trait bonus to one of these three skills, chosen when you take this trait.			CHOOSE:SKILL|Acrobatics,Climb,Swim		BONUS:SKILL|LIST|1|TYPE=Trait	CSKILL:LIST		SOURCEPAGE:p. 9	BONUS:SKILL|Acrobatics,Climb,Swim|min(max(floor(TL/3),1),-ACCHECK)												
```

Comment:
BONUS:SKILL|%LIST|1|TYPE=Trait - We avoid the LIST version in Bonus tags. CSKILL is appropriate, since it does not work with the %LIST version.
The second bonus looks interesting, I assume it is designed to counter the ACCHECK

Reply:
I don't see any MULT:YES tags with the choose tags. STACK is optional, but MULT:YES is required.